aksel.nav.no/website/pages/eksempler/tooltip/placement.tsx

```diff
@@ -10,17 +10,17 @@ import { withDsExample } from "@/web/examples/withDsExample";
 const Example = () => {
   return (
     <div className="grid gap-4">
-      <Tooltip content="Skriv ut dokument">
-        <Button icon={<ArrowUpIcon title="demo knapp" />} />
+      <Tooltip content="Skriv ut dokument" describesChild>
+        <Button icon={<ArrowUpIcon aria-hidden />} />
       </Tooltip>
-      <Tooltip content="Skriv ut dokument" placement="right">
-        <Button icon={<ArrowRightIcon title="demo knapp" />} />
+      <Tooltip content="Skriv ut dokument" placement="right" describesChild>
+        <Button icon={<ArrowRightIcon aria-hidden />} />
       </Tooltip>
-      <Tooltip content="Skriv ut dokument" placement="bottom">
-        <Button icon={<ArrowDownIcon title="demo knapp" />} />
+      <Tooltip content="Skriv ut dokument" placement="bottom" describesChild>
+        <Button icon={<ArrowDownIcon aria-hidden />} />
       </Tooltip>
-      <Tooltip content="Skriv ut dokument" placement="left">
-        <Button icon={<ArrowLeftIcon title="demo knapp" />} />
+      <Tooltip content="Skriv ut dokument" placement="left" describesChild>
+        <Button icon={<ArrowLeftIcon aria-hidden />} />
       </Tooltip>
```

Comment:
Looks like i flipped it when added it to demoes 🙈 I updated defaultValue to `false` since this will be a breaking change anyways, and updated the examples 👍

Reply:
Did you forget to update aksel.nav.no/website/pages/eksempler/togglegroup/with-tooltip.tsx?

Reply:
Messed up the rebase 💥